src/pages/Settings/index.js

```diff
@@ -119,100 +120,136 @@ const NotificationSettings = (
-                  <td>
-                    <input
-                      type="checkbox"
-                      name="showTreeToast"
-                      id="showTreeToast-input"
-                      checked={showTreeToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-                <tr>
-                  <td>Check connectivity</td>
-                  <td>
-                    <input
-                      type="checkbox"
-                      name="showConnectionStatusToast"
-                      id="showConnectionStatusToast-input"
-                      checked={showConnectionStatusToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-                <tr>
-                  <td>Use CTRL and scroll to zoom</td>
-                  <td>
-                    <input
-                      type="checkbox"
-                      name="showCtrlToZoomToast"
-                      id="showCtrlToZoomToast-input"
-                      checked={showCtrlToZoomToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-                <tr>
-                  <td>Customise Settings</td>
-                  <td>
-                    <input
-                      type="checkbox"
-                      name="showSettingsToast"
-                      id="showSettingsToast-input"
-                      checked={showSettingsToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-                <tr>
-                  <td>Help Links</td>
-                  <td>
-                    <input
-                      type="checkbox"
-                      name="showHelpToast"
-                      id="showHelpToast-input"
-                      checked={showHelpToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+) => {
+  const [status /*: string */, setStatus] = useState(
+    Notification.permission === 'granted' ||
+      Notification.permission === 'denied'
+      ? 'answered'
+      : 'not yet',
+  );
+  return (
+    <form data-category="notifications">
+      <h4>Notification settings</h4>
+      <SchemaOrgData
+        data={{
+          name: 'Notification settings',
+          description: 'Notifications in the website can be turned on/off.',
+        }}
+        processData={schemaProcessDataPageSection}
+      />
+      <div className={f('row')}>
+        <div className={f('medium-12', 'column')}>
+          {status === 'answered' ? (
+            <>
+              <p>
+                To change notification preference, please do in the browser
+                settings.
+              </p>
+              <button className={f('button')} disabled>
```

Comment:
I made quite a few changes on this part. Now I have made it an infobox to show the message. Please have a look! If we could settle on the message for both cases, it would be great